extensions/amp-bind/0.1/bind-expression.js

```diff
@@ -83,8 +93,12 @@ export class BindExpression {
     /** @const {string} */
     this.expressionString = expressionString;
 
-    /** @const {!./bind-expr-defines.AstNode} */
+    /** @const @private {!./bind-expr-defines.AstNode} */
     this.ast_ = parser.parse(this.expressionString);
+
+    /** @const @private {!Object} */
+    this.builtInScope_ = Object.create(null);
+    this.builtInScope_['Math'] = Math;
```

Comment:
So if someone names their state "Math", it will be override without warning with this?

I wonder if we should put all builtins under a single namespace and prevent that to be used as a top level key for state. `builtins.Math.foo`?  `AMP.Math.foo`? ...